src/group_theory/quotient_group.lean

```diff
@@ -315,6 +315,34 @@ monoid_hom.to_mul_equiv
   (by { ext ⟨x, hx⟩, refl })
   (by { ext ⟨x, hx⟩, refl })
 
+section zpow
+
+variables {G' H' : Type u} [comm_group G'] [comm_group H']
+variables (φ' : G' →* H') (ψ' : H' →* G') (χ : G' ≃* H')
+
+/-- The map of quotients by powers of an integer induced by a group homomorphism. -/
+@[to_additive "The map of quotients by multiples of an integer induced by an additive group
+homomorphism."]
+def hom_quotient_zpow_of_hom (n : ℤ) :
+  G' ⧸ (zpow_group_hom n : G' →* G').range →* H' ⧸ (zpow_group_hom n : H' →* H').range :=
+lift _ ((mk' _).comp φ') $
+  λ g ⟨h, (hg : h ^ n = g)⟩, (eq_one_iff _).mpr ⟨_, by simpa only [← hg, map_zpow]⟩
+
+@[to_additive, simp]
+lemma hom_quotient_zpow_of_hom_right_inverse (h : function.right_inverse ψ' φ') (n : ℤ) :
+  (hom_quotient_zpow_of_hom φ' n).comp (hom_quotient_zpow_of_hom ψ' n) = monoid_hom.id _ :=
```

Comment:
Is this true more generally as
```suggestion
  (hom_quotient_zpow_of_hom φ' n).comp (hom_quotient_zpow_of_hom ψ' n) = 
    hom_quotient_zpow_of_hom (φ'.comp ψ') n :=
```

Reply:
Oh, yo're right, this can be useful. Since this is already on the queue, @Multramate can you open another PR with this generalization? Thanks!